quickstarts/rhdh-installation-via-helm.yaml

```
@@ -194,7 +194,7 @@ spec:
           2. Click on the URL decorator in the top-right corner of the "developer-hub" Deployment to open your Red Hat Developer Hub instance.
           3. Verify that the correct URL is used and the Red Hat Developer Hub installation works fine.
         failedTaskHelp: |-
-          Doublecheck your Helm Chart hostname and TLS configuration, and if needed, the external DNS configuration.
+          Double-check your Helm Chart hostname and TLS configuration, and if needed, the external DNS configuration.
```

```suggestion
          Double check your Helm Chart hostname and TLS configuration, and if needed, the external DNS configuration.
```
Just needs a space, no hyphen.